src/charts/pie.coffee

```diff
@@ -57,8 +57,7 @@ Ember.Charts.PieComponent = Ember.Charts.ChartComponent.extend(
       value: d.value
       percent: d3.round( 100 * d.value / total )
 
-    key = @get 'sortKey'
-    data.sortBy key
+    _.sortBy data, @get('sortKey')
```

Comment:
nope.  i think someone have made Ember not modify the Array prototype? i'm not actually sure what's going on, but I am sure that `data` is not an Ember array